app/views/pages/_wip_page.html.erb

```diff
@@ -0,0 +1,8 @@
+<div class="content">
```

Comment:
Do we perhaps want to add this page to `.gitignore` so changes to it don't get written to Github? I like the idea of having a sample page in source control, but I'm concerned about constant updates to it. Perhaps something like `sample_application.yml` (we tell users to copy that to `application.yml`, and that file is in *.gitignore)